about/system_requirements.rst

```diff
@@ -23,15 +23,16 @@ Desktop or laptop PC - Minimum
 .. which can run up to macOS 10.13.
 
 +----------------------+-----------------------------------------------------------------------------------------+
-| **CPU**              | - **Windows:** x86_32 CPU with SSE2 instructions, x86_64 CPU, ARMv8 CPU                 |
+| **CPU**              | - **Windows:** x86_32 CPU with SSE2 support, x86_64 CPU with SSE4.2 support, ARMv8 CPU  |
 |                      |                                                                                         |
 |                      |   - *Example: Intel Core 2 Duo E8200, AMD Athlon XE BE-2300, Snapdragon X Elite*        |
```

Comment:
This AMD Athlon processor needs to be changed here and the 3 other places it shows up. From what I'm seeing on techpowerup [here](https://www.techpowerup.com/cpu-specs/athlon-x2-be-2300.c484), and on wikipedia [here](https://en.wikipedia.org/wiki/Athlon_64_X2), it doesn't support SSE4.2.